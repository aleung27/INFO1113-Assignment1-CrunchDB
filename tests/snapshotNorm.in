SET j 9
SNAPSHOT
SET b 1111111
SNAPSHOT
LIST SNAPSHOTS
ROLLBACK 1
SNAPSHOT
LIST SNAPSHOTS
LIST ENTRIES
SNAPSHOT
ROLLBACK 1
LIST SNAPSHOTS
LIST ENTRIES
