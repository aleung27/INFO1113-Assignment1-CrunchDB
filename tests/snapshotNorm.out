> ok

> saved as snapshot 1

> ok

> saved as snapshot 2

> 2
1

> ok

> saved as snapshot 2

> 2
1

> j [9]

> saved as snapshot 3

> ok

> 1

> j [9]

